Lib/multiprocessing/shared_memory.py

```diff
@@ -115,57 +113,146 @@ def __init__(self, name, flags=None, mode=384, size=0, read_only=False):
             except OSError as ose:
                 name_collision = False
             if name_collision:
-                raise ExistentialError(
-                    f"Shared memory already exists with name={name}"
-                )
+                raise FileExistsError(name)
```

Comment:
You could enrich the exception:

```python
raise FileExistsError(errno.EEXIST, os.strerror(errno.EEXIST), name)
```

Result:

```
FileExistsError: [Errno 17] File exists: 'filename'
```

Reply:
Good point -- changed in 68785333a25d902a42252ace3a8268357efd6094.